src/main/java/Deadline.java

```diff
@@ -0,0 +1,28 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+public class Deadline extends Tasks {
+    private LocalDate deadline;
+
+
+
```

Comment:
I think it would be nice to have only one blank line to separate the variables,methods  or logical units.